lua/lvim/core/alpha.lua

```diff
@@ -23,6 +23,9 @@ local function resolve_buttons(theme_name, entries)
     local button_element = selected_theme.button(entry[1], entry[2], entry[3])
     -- this became necessary after recent changes in alpha.nvim (06ade3a20ca9e79a7038b98d05a23d7b6c016174)
     button_element.on_press = on_press
+    if opts then
+      button_element.opts = vim.tbl_extend("force", button_element.opts, opts)
+    end
```

Comment:
```suggestion
```

Reply:
I put the default_button_opts in lvim.builtin.alpha.<name> like [this](https://github.com/LunarVim/LunarVim/pull/3532/files#diff-07600a29e85e57bafa844d558b3c260883e4b7c40d7e8f747aa18c3d73e45628R10) to make it easily configurable by the user. is that ok?

Reply:
see https://github.com/LunarVim/LunarVim/pull/3532#discussion_r1033899676